platforms/core-configuration/kotlin-dsl-tooling-models/build.gradle.kts

```diff
@@ -1,5 +1,5 @@
 plugins {
-    id("gradlebuild.distribution.api-kotlin")
+    id("gradlebuild.distribution.api-java")
```

Comment:
💭 This is to make sure we publish the API based on the Java classes. There is a single Kotlin class that is not required to be part of the published API.